mgmtsystem_audit/__manifest__.py

```diff
@@ -22,5 +22,5 @@
         "wizard/copy_verification_lines.xml",
     ],
     "demo": ["demo/demo_audit.xml"],
-    "development_status": "Production/Stable",
+    "installable": True,
```

Comment:
This is not necessary

Reply:
ok I change this

Reply:
Aha! so this is why you removed it.
The CI bot that checks for dev status inconsistencies is complaining:

```
Run manifestoo -d . check-dev-status --default-dev-status=Beta
mgmtsystem_audit (Production/Stable) depends on mgmtsystem_nonconformity (Beta)
```

Basically, we can't have Production/Stable modules depend on Beta modules.

I think we should bump `mgmtsystem_nonconformity` to Production/Stable instead, though. It's time.

Reply:
Hello @ivantodorovich 
Ok thanks what should I do ?

Reply:
Actually, on a second thought.. 

The dependency chain is a nightmare, and none of the depended modules have the `development_status` set on manifest. For this module to be Production/Stable, we need all other modules in the dependency chain to be Production/Stable as well. And these aren't just modules on this repo, also on knowledge repo (`document_page`)

I vote you simply go back to removing the `development_status` key on this PR. We can see later about bumping all of them, starting from the ones in knowledge repository

Reply:
@ivantodorovich 
Ok I go back to the previous development and it show green now.
Thank.